openid-sharedsignals-framework-1_0.md

```diff
@@ -951,6 +953,7 @@ Configuration ({{stream-config}}) object:
 * `events_requested`
 * `delivery` : Note that in the case of the poll method, the `endpoint_url` value is
   supplied by the Transmitter.
+* `description`
```

Comment:
This line looks strange. Should there be an explanation after this?

Reply:
It should not have an explanation, but I can move the note that is attached to `delivery` so that it is below the list.

Reply:
I believe this is a normative change. Any reason to think of this as a non-normative change?